apps/payments/app/[locale]/(hosted)/paymentRequest/utils.ts

```
@@ -7,7 +7,7 @@ export async function getTokenPayload(token: string) {
   const paymentsApi = await AuthenticationFactory.getPaymentsClient();
   const { data: payload, error } = await paymentsApi.decodeToken({ token });
 
-  if (error) errorHandler(error);
+  if (error) console.log("error decoding token: ", error);
```

Is this a leftover, or do we want to keep the console.log?

---

In the ticket description it says we want to use the default redirect URL without interruption. So we need to ignore errors when decoding the token, but I thought we might want to log the error so to have visibility

---

Good catch! Thanks for explaining